constants.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   INDEX_HEAD: '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF',
   ADDRESS_ZERO: '0x0000000000000000000000000000000000000000',
   DEFAULT_CONFIRMATIONS: 2,
-  ERC20_INTERFACE_ID: '0x277f8169',
+  ERC20_INTERFACE_ID: '0x36372b07',
```

Comment:
Instead of these constants being copy pasted they could just be imported from order-utils, would prevent bugs like this in the future 
https://github.com/airswap/airswap-protocols/blob/master/utils/order-utils/src/constants.js#L29